parsing/pcoq.ml

```diff
@@ -98,7 +98,7 @@ module EntryDataMap = EntryCommand.Map(EntryData)
 
 type ext_kind =
   | ByGrammar of extend_rule
-  | ByEXTEND of (unit -> unit) * (unit -> unit)
+  | ByEXTEND of (unit -> unit) * (unit -> unit) option
```

Comment:
As explained in the commit message this whole file diff is just a hack for GRAMMAR EXTEND and should be replaced by something better.